source/partials/next_release/_release.md.erb

```diff
@@ -1,15 +1,29 @@
 #### API Enhancements
 
-* <%= link_to_commit 'f47d054171b07e2ce9c2efa9e02dde6979fc9c8f', "#{link_to_api 'encryption'} to encrypt secrets.", 1829 %>
+- <%= link_to_commit '73595c1d18f7455c5b7aac39dce319c987f0c231', 'Check Package Connection API (#3048)', 3048 %>
+- <%= link_to_commit '86729312550d47c7268c615fc0926bd6d37d4d7e', 'Check Repository Connection API. (#2953)', 2954 %>
+- <%= link_to_commit '041572cf78caecb32157d7a68fdca183dcff6674', 'Clone notification API endpoint.', 2954 %>
+- <%= link_to_commit '2789aa8f691a860c7908aabb42cacc99a7333031', 'Allow template admin to view and edit templates via API.', 2967 %>
+- <%= link_to_commit '3c001b1d376263e964495723fb76a4771ca8a705', 'API V2 of stage status notification.', 3030 %>
```

Comment:
This is not an api change. The commit said it was version 2 of the notification endpoint. This is for the notification plugins.